### PR TITLE
Released versions of Visual Studio 2022 links

### PR DESCRIPTION
**Pull request by NimaHp**
Released versions of Visual Studio 2022 are available

### CODE REVIEW

**Comment by PRMerger8** [comment]
@NimaHp : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: docs/install/create-an-offline-installation-of-visual-studio.md [hunk 1]
```diff
@@ -103,14 +103,13 @@ Start by downloading the Visual Studio 2019 bootstrapper from either the [Visual
...
->[!TIP]
-> Released versions of Visual Studio 2022 are not yet available, the bootstrappers below are for the preview release of Visual Studio 2022.
->Start by downloading the Visual Studio 2022 bootstrapper from the [Visual Studio downloads page](https://aka.ms/vs2022preview).
+
 
 | Edition                         | Download                                                            |
 |---------------------------------|---------------------------------------------------------------------|
-| Visual Studio 2022 Professional | [vs_professional.exe](https://aka.ms/vs/17/pre/vs_professional.exe) |
-| Visual Studio 2022 Enterprise   | [vs_enterprise.exe](https://aka.ms/vs/17/pre/vs_enterprise.exe)     |
+| Visual Studio 2022 Community | [vs_professional.exe](https://aka.ms/vs/17/release/vs_community.exe) |
```
Comment (anandmeg) [inline docs/install/create-an-offline-installation-of-visual-studio.md:110]: | Visual Studio 2022 Community | [vs_professional.exe] should say | Visual Studio 2022 Community | [vs_**community**.exe]

File: docs/install/create-an-offline-installation-of-visual-studio.md [hunk 2]
```diff
@@ -103,14 +103,13 @@ Start by downloading the Visual Studio 2019 bootstrapper from either the [Visual
 
 ::: moniker range=">=vs-2022"
 
->[!TIP]
-> Released versions of Visual Studio 2022 are not yet available, the bootstrappers below are for the preview release of Visual Studio 2022.
->Start by downloading the Visual Studio 2022 bootstrapper from the [Visual Studio downloads page](https://aka.ms/vs2022preview).
+
```
Comment (anandmeg) [inline docs/install/create-an-offline-installation-of-visual-studio.md:106]: Let's keep the following text: 
Start by downloading the Visual Studio 2022 bootstrapper from the [Visual Studio downloads page](https://visualstudio.microsoft.com/downloads).

**Review by anandmeg — CHANGES_REQUESTED** [review]
@NimaHp, thank you for your contribution in keeping our page up-to-date.

**Comment by PRMerger13** [comment]
@NimaHp : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/install/create-an-offline-installation-of-visual-studio.md (hunk 1 above)
Comment (NimaHp) [reply docs/install/create-an-offline-installation-of-visual-studio.md]: Sorry Missed it.

**Review by anandmeg — APPROVED** [review]
@NimaHp, thank you again for contributing to our docs.

**Comment by PRMerger8** [comment]
@NimaHp : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/install/create-an-offline-installation-of-visual-studio.md [hunk 3]
```diff
@@ -71,12 +71,12 @@ To get the latest bootstrappers for Visual Studio 2019 that will always install
...
 
-| Edition                         | Bootstrapper                                                            |
-|---------------------------------|---------------------------------------------------------------------|
-| Visual Studio 2022 Community | [vs_community.exe](https://aka.ms/vs/17/release/vs_community.exe) |
-| Visual Studio 2022 Professional | [vs_professional.exe](https://aka.ms/vs/17/pre/vs_professional.exe) |
-| Visual Studio 2022 Enterprise | [vs_enterprise.exe](https://aka.ms/vs/17/pre/vs_enterprise.exe)     |
-| Visual Studio 2022 Professional | [vs_buildtools.exe](https://aka.ms/vs/17/pre/vs_buildtools.exe) |
+| Edition                         | Download                                                                |
```
Comment (anandmeg) [inline docs/install/create-an-offline-installation-of-visual-studio.md:74]:
```suggestion
| Edition                         | Bootstrapper                                                               |
```

File: docs/install/create-an-offline-installation-of-visual-studio.md [hunk 4]
```diff
@@ -71,12 +71,12 @@ To get the latest bootstrappers for Visual Studio 2019 that will always install
...
+|---------------------------------|-------------------------------------------------------------------------|
+| Visual Studio 2022 Community    | [vs_community.exe](https://aka.ms/vs/17/release/vs_community.exe)       |
+| Visual Studio 2022 Professional | [vs_professional.exe](https://aka.ms/vs/17/release/vs_professional.exe) |
+| Visual Studio 2022 Enterprise   | [vs_enterprise.exe](https://aka.ms/vs/17/release/vs_enterprise.exe)     |
+| Visual Studio 2022 Build Tools   | [vs_buildtools.exe](https://aka.ms/vs/17/pre/vs_buildtools.exe)         |
```
Comment (anandmeg) [inline docs/install/create-an-offline-installation-of-visual-studio.md:79]:
```suggestion
| Visual Studio 2022 Build Tools   | [vs_buildtools.exe](https://aka.ms/vs/17/release/vs_buildtools.exe)         |
```

**Review by anandmeg — CHANGES_REQUESTED** [review]

**Comment by PRMerger19** [comment]
@anandmeg : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@anandmeg : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by anandmeg — APPROVED** [review]

**Comment by anandmeg** [comment]
#sign-off

**Pull request merged**